gui/directoryservice/forms.py

```diff
@@ -243,6 +217,21 @@ class ActiveDirectoryForm(ModelForm):
         label=_("NetBIOS alias"),
         required=False,
     )
+    ad_kerberos_principal = forms.ChoiceField(
+        label=models.ActiveDirectory._meta.get_field('ad_kerberos_principal').verbose_name,
+        required=False,
+        choices=choices.KERBEROS_PRINCIPAL_CHOICES(),
+        help_text=_(
+            "Kerberos principal to use for AD-related UI and middleware operations "
+            "Field is populated with principals present in the system keytab. "
+            "During the domain join process a keytab entry is generated for the "
+            "AD Machine Account associated with the NAS. The name for this account "
+            "is the netbios name of the server with a '$' appended to it. Once "
+            "the NAS is joined to active directory, the bind credentials will be "
+            "automatically cleared and all future operations carried out by the AD "
+            "machine account, which has a restricted set of privileges in the AD domain."),
```

Comment:
"Once the NAS is joined to active directory, the bind credentials will be automatically cleared and all future operations carried out by the AD machine account, which has a restricted set of privileges in the AD domain." --> 

"Bind credentials are automatically cleared after the NAS is joined to active directory. Later operations are done by the AD machine account, which has restricted privileges in the AD domain."